.tekton/osc-podvm-builder-pull-request.yaml

```diff
@@ -188,40 +181,40 @@ spec:
         values:
         - "true"
       workspaces:
-      - name: output
-        workspace: workspace
       - name: basic-auth
         workspace: git-auth
     - name: prefetch-dependencies
       params:
       - name: input
         value: $(params.prefetch-input)
-      - name: dev-package-managers
-        value: "true"
```

Comment:
This one is a bit hidden, but we also need to put it back.
It is a requirement for RPM pre-fetch